src/tests/accomodation/accommodation.test.js

```diff
@@ -103,3 +110,58 @@ describe('Test getting accommodations:', () => {
       });
   });
 });
+
+describe('Test like/unlike accommodation:', () => {
+  before(async () => {
+    await createNewBooking();
+  });
+  it('Should allow user to like accommodation facility', (done) => {
+    chai.request(app)
+      .patch(`/api/accommodations/${newBooking.accommodationId}/like-unlike`)
+      .set('Authorization', loggedInToken)
+      .send(accommodationLikeValue)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.should.have.status(200);
+        res.body.should.have.property('message').equal('Operation perfomed successfully');
+        done();
+      });
+  });
+  it('Should allow user to unlike accommodation facility', (done) => {
+    chai.request(app)
+      .patch(`/api/accommodations/${newBooking.accommodationId}/like-unlike`)
+      .set('Authorization', loggedInToken)
+      .send(accommodationUnlikeValue)
+      .end((err, res) => {
+        res.body.should.be.an('object');
+        res.should.have.status(200);
+        res.body.should.have.property('message').equal('Operation perfomed successfully');
+        done();
+      });
+  });
+  it('Should allow NOT user to Like or Unlike accommodation facility: Booking Not Found', (done) => {
```

Comment:
@higustave-ops Double check the grammar. should allow not? should not allow.
Again, the test case description is not describing the behavior or functionality being tested.